xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/uicomponents/form/form.css

```diff
@@ -0,0 +1,7 @@
+/* Visually convey that the form is being submitted once a first submission was done on a form. */
+form.xwiki-form-submitting,
+form.xwiki-form-submitting input,
+form.xwiki-form-submitting select,
+form.xwiki-form-submitting button {
+  cursor: wait;
```

Comment:
Currently the only visual clue is the spinning cursor. Feels enough to me but let me know if you think we should improve this.

Reply:
This means there won't be any visible feedback on touchscreen devices which doesn't sound good.

Reply:
Hum indeed. I'll look for another visual clue that does not involve a cursor.